cdi-test-core/src/test/java/de/hilling/junit/cdi/AcitvateAlternativeForAlternativeTest.java

```diff
@@ -0,0 +1,52 @@
+package de.hilling.junit.cdi;
+
+import de.hilling.junit.cdi.beans.Person;
+import de.hilling.junit.cdi.service.BackendService;
+import de.hilling.junit.cdi.service.OverriddenService;
+import de.hilling.junit.cdi.service.TestActivatedOverridenService;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Inject;
+
+@RunWith(CdiUnitRunner.class)
+public class AcitvateAlternativeForAlternativeTest {
```

Comment:
Jau!
Mit freundlichen Grüßen,

Gunnar Hilling

Dipl.-Phys.
Geschäftsführer

Gunnar Hilling IT Consulting GmbH

Telefon	+49 541 4068204
e-mail	gunnar@hilling.de <mailto:gunnar@hilling.de>

Geschäftsführender Gesellschafter: Gunnar Hilling
Registergericht: Amtsgericht Osnabrück
Handelsregister: HRB 206478

> Am 28.10.2015 um 23:05 schrieb fseemann <notifications@github.com>:
> 
> In cdi-test-core/src/test/java/de/hilling/junit/cdi/AcitvateAlternativeForAlternativeTest.java <https://github.com/guhilling/cdi-test/pull/29#discussion_r43324994>:
> 
> > @@ -0,0 +1,52 @@
> > +package de.hilling.junit.cdi;
> > +
> > +import de.hilling.junit.cdi.beans.Person;
> > +import de.hilling.junit.cdi.service.BackendService;
> > +import de.hilling.junit.cdi.service.OverriddenService;
> > +import de.hilling.junit.cdi.service.TestActivatedOverridenService;
> > +import org.junit.Assert;
> > +import org.junit.Test;
> > +import org.junit.runner.RunWith;
> > +
> > +import javax.inject.Inject;
> > +
> > +@RunWith(CdiUnitRunner.class)
> > +public class AcitvateAlternativeForAlternativeTest {
> Typo in class name.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/guhilling/cdi-test/pull/29/files#r43324994>.
>